appside/nav/nav.js

```diff
@@ -103,16 +103,16 @@ class FrameFactory {
      */
     static build(frame) {
         // TODO: more sane identification of frame type (type property)
-        if(frame.hasOwnProperty('text')) {
+        if(frame.template === INTRO_FRAME_TEMPLATE) {
             return new IntroFrame(frame);
-        }
-        else if(frame.hasOwnProperty('statements')) {
+        } else if(frame.template === STATEMENTS_FRAME_TEMPLATE) {
             // TODO: identify when a different subtype of body frame is needed
             return new StatementsBodyFrame(frame);
-        }
-        else if(frame.hasOwnProperty('matched_emotions')) {
+        } else if(frame.template === SUMMARY_COUNT_FRAME_TEMPLATE) {
             // TODO: identify when this should be qualifier
             return new SummaryFrameCount(frame);
+        } else if(frame.template === LIKERT_FRAME_TEMPLATE) {
+            return new LikertFrame(frame);
         }
```

Comment:
Add
```
} else {
    throw new Error('Frame template not recognized ' + frame.template);
}
```
This will prevent some of the more mysterious errors we were getting due to the wrong frame type being instantiated.

Reply:
Added. Good checkpoint for debugging.